install.sh

```diff
@@ -260,6 +273,38 @@ set_synchrony_url() {
   fi
 }
 
+# Update the current load balancer listener on port 7999 to use the TCP protocol
+enable_tcp_protocol_on_lb_listener() {
+  local install_bitbucket
+  local region
+  local load_balancer_dns
+  local load_balancer_name
+  local original_instance_port
+
+  install_bitbucket=$(get_product "bitbucket" "${CONFIG_ABS_PATH}")
+
+  if [ -n "${install_bitbucket}" ]; then
+    region=$(get_variable 'region' "${CONFIG_ABS_PATH}")
+    load_balancer_dns=$(terraform output | grep '"load_balancer_hostname" =' | sed -nE 's/^.*"(.*)".*$/\1/p')
```

Comment:
I see we use `terraform output` in more than once in installer script (right before tagging process, when terraform apply is done). As running this command takes some time, and the result of multiple run is similar, maybe is good to run this command once and keep it in a variable in script and reuse it whenever is needed to improve the performance a bit.

Reply:
I couldn't find the code change for this comment. Or maybe we can optimise it as part of another PR. In this case good to raise a ticket to track this improvement.

Reply:
👍🏽  Created this: https://bulldog.internal.atlassian.com/browse/TFKUBE-368